src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,7 +1,190 @@
+    private static final String SQL_URL = "localhost:5432/";
+    private static final String DATABASE_NAME = "sqlcmd";
+    private static final String USER_NAME = "sqlcmd";
+    private static final String USER_PASSWORD = "sqlcmd";
+    private static final String LINE_SEPARATOR = System.getProperty("line.separator");
+    private static final String USER_TABLE_NAME = "\"user\"";
+    private Connection connection = null;
+
+    Application() throws SQLException, ClassNotFoundException {
+        connection = getConnection();
+    }
+
+    public static void main(String[] args) throws SQLException, ClassNotFoundException {
+        new Application().simpleSQL();
+
+    }
+
+    public void simpleSQL() {
+        printTableList();
+
+        dropTableIfExists();
+        printTableList();
+
+        createTableUser();
+        printTable(USER_TABLE_NAME);
+        printTableList();
+
+        insertIntoUserTable("user1", "password1");
+        insertIntoUserTable("user2", "password2");
+        insertIntoUserTable("user3", "password3");
+        printTable(USER_TABLE_NAME);
+
+        changePasswordInUserTable("user1", "password2");
+        printTable(USER_TABLE_NAME);
+
+        removeFromUserTable("user3");
+        printTable(USER_TABLE_NAME);
+
+        if (connection != null) {
+            try {
+                connection.close();
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    private void printStatus(String operation, String subject) {
+        System.out.println(operation + " table " + subject + "..");
+    }
+
+    private void dropTableIfExists() {
+        String sqlQuery = "DROP TABLE IF EXISTS " + USER_TABLE_NAME;
+        printStatus("Removing", USER_TABLE_NAME);
+        int affectedRows = executeUpdateQuery(sqlQuery);
+        printAffectedRows(affectedRows);
+    }
+
+    private void changePasswordInUserTable(String name, String password) {
+        String sqlQuery = String.format("UPDATE " + USER_TABLE_NAME + " SET password = '%s'" + " WHERE name='%s'", password, name);
+        printStatus("Updating", USER_TABLE_NAME);
+        int affectedRows = executeUpdateQuery(sqlQuery);
+        printAffectedRows(affectedRows);
+    }
+
+    private void removeFromUserTable(String name) {
+        String sqlQuery = String.format("DELETE FROM " + USER_TABLE_NAME + " WHERE name='%s'", name);
+        int affectedRows = executeUpdateQuery(sqlQuery);
+        printAffectedRows(affectedRows);
+    }
+
+    private void printTable(String tableName) {
+        String sqlQuery = String.format("SELECT * FROM %s order by %<s.id", USER_TABLE_NAME);
+        try (PreparedStatement statement = connection.prepareStatement(sqlQuery)) {
+            printStatus("Printing", tableName);
+            statement.execute();
+            ResultSet resultSet = statement.getResultSet();
+
+            if (!resultSet.isBeforeFirst()) {
```

Comment:
Maybe better write like this:
if (resultSet.isBeforeFirst()) {
...
} else { 
System.out.println("table is empty" + LINE_SEPARATOR);
}
What do you think?